### PR TITLE
SerializerAppend2: fix nextPow2 and buffer size increase logic

### PR DESCRIPTION
**Pull request by mcintyret**

### CODE REVIEW

File: lib/SerializerAppend2.js [hunk 1]
```diff
@@ -16,7 +16,11 @@ const rename = promisify(fs.rename);
 const rimraf = promisify(_rimraf);
 const write = promisify(fs.writeFile);
 
-const nextPow2 = n => Math.pow(2, Math.ceil(Math.log(n) / Math.log(2)));
```
Comment (mcintyret) [inline lib/SerializerAppend2.js]: The existing implementation had a bug where `nextPow2(n) === n` if n is a power of 2

**Comment by Que3216** [comment]
@mzgoddard This is an awesome plugin -- it looks like it'll speed up our subsequent builds by 3x. Do you know when you'll have time to review this?

**Comment by nomcopter** [comment]
Running into this as well, would love this fix.

**Comment by mzgoddard** [comment]
Merged as #374

**Pull request closed without merging**